### ISSUE TITLE
Selecting BaseRating.GREAT progammatically makes the view only partially visible on the left 

### ISSUE THREAD

**Issue by muigukenneth** [sujithkanna/SmileyRating#11]
When i use smileRating.setSelectedSmile(BaseRating.GREAT); it makes the view invisible(only partially visible on the left) but the rest of the rating work okay during the initialization of the view .When you tap on any rating it reverts to the normal state.Cheers.

**Comment by sujithkanna** [comment]
Can you attach the screenshot, So that I can clearly see the problem. Thank you :)

**Comment by muigukenneth** [comment]
![screenshot_2017-04-18-18-22-20](https://cloud.githubusercontent.com/assets/4389000/25138770/0fe31306-2464-11e7-9693-a90122e3b49b.png)
![screenshot_2017-04-18-18-22-06](https://cloud.githubusercontent.com/assets/4389000/25138771/0ff74f6a-2464-11e7-8aef-d3c0ee3cc1b5.png)
You can check out the first one(great) then the second (good) is okay :)

**Comment by sujithkanna** [comment]
Really sorry for this, can you tell me the device model and if possible can you share the xml layout you created?

**Comment by muigukenneth** [comment]
Here is the model and the xml file.Thanks :

 Android build version: G900HXXS1CPJ9
Android release version: 6.0.1
Android SDK version: 23
Android build ID: MMB29K.G900HXXS1CPJ9
Device brand: samsung
Device manufacturer: samsung
Device name: k3g
[fragment_about.txt](https://github.com/sujithkanna/SmileyRating/files/942352/fragment_about.txt)

**Comment by DemidovIlya** [comment]
I have the same problem

**Comment by sujithkanna** [comment]
Really sorry for the delay, came back after a long vacation. I couldn't find the problem. You are saying it is only happening on setting GREAT. Let's try one thing, try to set some size for the rating bar. Just set the width of the ratingbar. Height will be automatically taken care of. Let me know the output.

**Comment by DemidovIlya** [comment]
Yep, this helped me make it work correct. The problem appears when view width equals to display width. At first I just set width in layout file and this solve problem, so I tried to change width programmaticaly and when I set view width equals to display width it stop working again. For normal work, view width just should't equals to display width. 

```
Point size=new Point();
Display display=getWindowManager().getDefaultDisplay();
display.getSize(size);
mSmileRating.setLayoutParams(new LinearLayout.LayoutParams(size.x-1, ViewGroup.LayoutParams.WRAP_CONTENT));
```

Thanks a lot.
